doc/users/next_whats_new/colormaps.rst

```diff
@@ -1,8 +1,9 @@
-Colormap registry
-------------------
+Colormap registry (experimental)
+--------------------------------
 
 Colormaps are now managed via `matplotlib.colormaps`, which is a
-`.ColormapRegistry`.
+`.ColormapRegistry`. This API is experimental and may still change in future
+versions.
 
 Colormaps can be obtained using item access::
 
```

Comment:
I don't think we should discourage something in favour of something else that's experimental - perhaps worth removing the sentence starting on line 17 below? (I couldn't comment on a line lower than this)

Reply:
Given the examples are changing to `plt.colormaps`, change the example below this line too?

Reply:
> I don't think we should discourage something in favour of something else that's experimental

Formally I see the point. What is important to me is that given we now have two APIs for a transition time, we should give a clear recommendation to the users which ones they should prefer. It's quite annoying if you have multiple ways and have to decide yourself. In the present context, I advocate to use the new API for new code. The benefit of not having to switch the code to the new API later is larger than the risk that the API (in particular the most common feature of getting a colormap via `colormaps[name]`) will change. - To put it simply: Either you'll have to adapt your code later for sure, or there's a small chance that you have to adapt your code later.

Effectivelym "discouraged" is still a weak driver. It's only wording in the docs and does not show up as warning like a deprecation would. Would you be more comfortable with the wording

> We recommend to use the new API instead of `matplotlib.cm.get_cmap` and `matplotlib.cm.register_cmap`.